cmake/dependencies.cmake

```diff
@@ -304,11 +304,12 @@ if(ENABLE_INTEL_GNA)
         endif()
 
         RESOLVE_DEPENDENCY(GNA_EXT_DIR
-                ARCHIVE_UNIFIED "GNA/GNA_${GNA_VERSION}.zip"
+                ARCHIVE_UNIFIED "gna/GNA_${GNA_VERSION}.zip"
```

Comment:
Wouldn't it make sense to make the file name lowercase as well?

Reply:
I would keep it.
The file is already uploaded.

Reply:
It would, probably, but the file with uppercase name was already uploaded to the server As we can't delete anything from the server right now, a new archive would need to be uploaded. Imho, it makes little sense to have two exact versions of library on the server with the only difference being uppercase vs lowercase name - maybe we could change it with the next library version?

Reply:
If uploading isn't easy, than sure.